loops-and-arrays.js

```diff
@@ -57,5 +65,7 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 11 is 1331'
 // ]
 
-
+for (let i = 0; i < squares.length; i++) {
+    console.log((squares[i]*squares[i])*squares[i])
+}
 
```

Comment:
Je hebt het eind resultaat gevonden, goed gedaan, echter ben je de waarde van je begin variabel kwijt. Wat ik hieruit kan halen is dat je squares[arrray], de waarde daarin is veranderd naar de uitkomst. Een oplossing was geweest je berekening in een nieuwe variabel opslaan en uiteindelijk beide printen/loggen.